tools/docker/docker-compose.yml

```diff
@@ -49,3 +49,16 @@ services:
     environment:
       - HOST_PORT=${PORT_WORDPRESS:-80}
       - COMPOSE_PROJECT_NAME=$COMPOSE_PROJECT_NAME
+
+  monorepo:
```

Comment:
This breaks normal use of `jetpack docker up`.
```
$ jetpack docker up -d
WARNING: The HOST_CWD variable is not set. Defaulting to a blank string.
ERROR: build path /tools/docker either does not exist, is not accessible, or is not a valid URL.
Command exited with status 1
```

Reply:
I believe this should work now after e098e69

Reply:
```
$ jetpack docker up
ERROR: build path /usr/local/src/automattic/jetpack/tools/docker/tools/docker either does not exist, is not accessible, or is not a valid URL.
Command exited with status 1
```
Fixing that will, for me, likely still run into https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1087370. Although we probably don't need to consider me personally as a blocker.

But even for everyone else that it does work for, it'll mean that `jetpack docker up` will unnecessarily build that image to start an instance of the container which will then immediately exit.

Reply:
nod, thanks. I'll dig into it more than the quick-looking fix.